app/Service/HistoricalWeatherHumidityService.php

```diff
@@ -24,4 +33,22 @@ public function processHistoricalWeatherHumidityApiResponse( array $apiResponse
 
```

Comment:
i know it didn't change in this branch, but:
you instantiate a HistoricalWeatherHumidityModel, then you treat it as an array...
please change
`$historicalWeatherHumidity['longitude'] = $longitude; 
$historicalWeatherHumidity['latitude'] = $latitude;
$historicalWeatherHumidity['relative_humidity_2m'] = $humidity;
$historicalWeatherHumidity['date_time_of_measurement'] = $dateTimeOfMeasurement;`

to

`$historicalWeatherHumidity->longitude = $longitude; 
$historicalWeatherHumidity->latitude = $latitude;
$historicalWeatherHumidity->relative_humidity_2m = $humidity;
$historicalWeatherHumidity->date_time_of_measurement = $dateTimeOfMeasurement;`

Reply:
- [ ] fix Model property usage